scenes/menu/char_select.tscn

```diff
@@ -346,6 +347,15 @@ layout_mode = 2
 char_id = "nine"
 portrait_texture = ExtResource("34_h13ui")
 
+[node name="Row3" type="HBoxContainer" parent="CenterContainer/BBCharacterSelect/Rows"]
+layout_mode = 2
+alignment = 1
+
+[node name="Taokaka" parent="CenterContainer/BBCharacterSelect/Rows/Row3" instance=ExtResource("6_3lmwy")]
+layout_mode = 2
+char_id = "taokaka"
+portrait_texture = ExtResource("37_iqb1f")
+
```

Comment:
If you didn't already, it might be good to move 1-2 of the row two characters down here

Reply:
This is just my incentive to add more characters.